logfire/_internal/json_schema.py

```diff
@@ -68,15 +68,11 @@ def type_to_schema() -> dict[type[Any], JsonDict | Callable[[Any, set[int]], Jso
                 pydantic.NameEmail: {'type': 'string', 'x-python-datatype': 'NameEmail'},
                 pydantic.SecretStr: {'type': 'string', 'x-python-datatype': 'SecretStr'},
                 pydantic.SecretBytes: {'type': 'string', 'x-python-datatype': 'SecretBytes'},
+                pydantic.AnyUrl: {'type': 'string', 'x-python-datatype': 'AnyUrl'},
```

Comment:
Can decide on how many of these you want to add 👍

Reply:
```suggestion
                pydantic.AnyUrl: {'type': 'string', 'x-python-datatype': 'Url'},
```

hopefully that means the test changes can be reverted. the old tests make more sense because `repr(pydantic.AnyUrl('http://example.com'))` is `Url('http://example.com/')`.

Reply:
@alexmojaki, with `pydantic` `main`, `repr(pydantic.AnyUrl('http://example.com'))` is `AnyUrl('http://example.com')`, which is more intuitive

Reply:
Let's go with `Url` for now so that tests pass and we can merge this and deal with the other new test failures from the otel release, we can revisit afterwards

Reply:
i don't think the repr determines the output, it's just something we'd like to mimic